core/src/types/account.rs

```diff
@@ -49,7 +17,7 @@ impl Account {
     Deserialize,
     Default,
 )]
-/// Holds the public key map data as a bimap for efficient querying
+/// Holds the public key map data as a bimap for efficient quering
```

Comment:
This was actuallly correct before.

Reply:
sry, must have accidentally revived it at some point, fixed in [7c0364c](https://github.com/anoma/namada/pull/2312/commits/7c0364c44a3a59c1b8a02c221bf64eac7f5beb42)